mm/kfence/core.c

```diff
@@ -255,6 +255,32 @@ size_t kfence_ksize(const void *addr)
 	return abs(READ_ONCE(kfence_metadata[index].size));
 }
 
+static void kfence_report_corruption(unsigned long address);
```

Comment:
Can they be moved so this it not required?

Reply:
These reporting functions actually belong together, I don't think it'll look better if we move one of them to the top of the file.
I've been thinking about moving them away to `mm/kfence/report.c`, but that's not too much code now, so I'll just move this declaration to `core.h`